cohere/client.py

```diff
@@ -214,16 +221,38 @@ def chat(
         if chatlog_override is not None:
             self._validate_chatlog_override(chatlog_override)
 
+        if session_id != "":
+            conversation_id = session_id
+            warnings.warn(
```

Comment:
I think you can use `logger.warning` instead

Reply:
I'll update =D but just out of curiosity, should we use one over the other?

Reply:
so we can have more control over logging if needed.